Classes/TypeResolver.php

```diff
@@ -33,7 +33,7 @@ public function get($typeClassName)
         if (!is_string($typeClassName)) {
             throw new \InvalidArgumentException(sprintf('Expected string, got "%s"', is_object($typeClassName) ? get_class($typeClassName) : gettype($typeClassName)), 1460065671);
         }
-        if (!is_subclass_of($typeClassName, Type::class)) {
+        if (!is_subclass_of($typeClassName, Type::class, true)) {
```

Comment:
According to https://www.php.net/manual/en/function.is-subclass-of.php that's the default anyways. But it's nicer like this, so thanks for that :)